app/src/views/features/sessions/SessionViewer/DraftSessionViewer.tsx

```diff
@@ -144,7 +216,8 @@ const DraftSessionViewer: React.FC = () => {
       content: "Are you sure you want to discard this draft recording?",
       okText: "Yes",
       cancelText: "No",
-      onOk() {
+      onOk: async () => {
+        await handleClearDraftSessionCache();
```

Comment:
Do we need `await` here? It can be done asynchronously once we move the `clearDraftSessionCache` into separate module then the function would take `tabId` as an argument.